examples/svm/plot_separating_hyperplane.py

```diff
@@ -12,37 +12,39 @@
 import numpy as np
 import matplotlib.pyplot as plt
 from sklearn import svm
-
+from sklearn.datasets import make_classification
 # we create 40 separable points
 np.random.seed(0)
-X = np.r_[np.random.randn(20, 2) - [2, 2], np.random.randn(20, 2) + [2, 2]]
-Y = [0] * 20 + [1] * 20
 
+X, Y = make_classification(n_features=2, n_redundant=0, n_informative=1,
+                           n_clusters_per_class=1)
 # fit the model
 clf = svm.SVC(kernel='linear')
 clf.fit(X, Y)
 
-# get the separating hyperplane
-w = clf.coef_[0]
-a = -w[0] / w[1]
-xx = np.linspace(-5, 5)
-yy = a * xx - (clf.intercept_[0]) / w[1]
-
-# plot the parallels to the separating hyperplane that pass through the
-# support vectors
-b = clf.support_vectors_[0]
-yy_down = a * xx + (b[1] - a * b[0])
-b = clf.support_vectors_[-1]
-yy_up = a * xx + (b[1] - a * b[0])
-
-# plot the line, the points, and the nearest vectors to the plane
-plt.plot(xx, yy, 'k-')
-plt.plot(xx, yy_down, 'k--')
-plt.plot(xx, yy_up, 'k--')
-
-plt.scatter(clf.support_vectors_[:, 0], clf.support_vectors_[:, 1],
-            s=80, facecolors='none')
-plt.scatter(X[:, 0], X[:, 1], c=Y, cmap=plt.cm.Paired)
+plt.scatter(X[:, 0], X[:, 1], c=Y, s=50, cmap='autumn', edgecolors='k')
```

Comment:
Do not change something unless there is a very good reason to. In this case, I would keep the `plt.cm.Paired` colormap.